extensions/search-npm/package.json

```
@@ -0,0 +1,38 @@
+{
+  "name": "search-npm",
+  "title": "Search npm",
```

The title is used in the store. I would suggest to use something like this.

```suggestion
  "title": "npm Package Search",
```